services/orchest-webserver/client/src/hooks/useCheckUpdate.tsx

```diff
@@ -41,15 +41,19 @@ const fetchOrchestVersion = () =>
     (response) => response.version
   );
 
-const fetchLatestVersion = () =>
-  fetcher<UpdateInfo>("/async/orchest-update-info").then(
+const fetchLatestVersion = (cache?: boolean) => {
+  const endpoint = `/async/orchest-update-info?cache=${
+    cache ? "true" : "false"
+  }`;
```

Comment:
Building on the previous comment: By providing default value for `cache` you can just interpolate it in the string directly:

```suggestion
  const endpoint = `/async/orchest-update-info?cache=${cache}`;
```